src/PostgREST/Query/Statements.hs

```diff
@@ -10,7 +10,9 @@ This module constructs single SQL statements that can be parametrized and prepar
 TODO: Currently, createReadStatement is not using prepared statements. See https://github.com/PostgREST/postgrest/issues/718.
```

Comment:
This looks like it is solved, wdyt?

Reply:
Hm, not sure - it doesn't look like we are explicitly using prepared statements here. @steve-chavez did we change this since that comment?

Reply:
It was my impression that we are using prepared statements everywhere by now.

Reply:
Yeah, this is solved, the comment is a leftover.

https://github.com/PostgREST/postgrest/blob/082c91c85518d4b2ea97b848bba6e2981d962743/src/PostgREST/Query/Statements.hs#L92-L95

[dynamicallyParameterized](https://hackage.haskell.org/package/hasql-dynamic-statements-0.3.1/docs/Hasql-DynamicStatements-Statement.html#v:dynamicallyParameterized) takes a Bool argument for preparing the statement.

Which is set here(below, needs scrolling) with `configDbPreparedStatements`:

https://github.com/PostgREST/postgrest/blob/082c91c85518d4b2ea97b848bba6e2981d962743/src/PostgREST/App.hs#L237-L250